frontend/packages/dev-console/src/components/topology-list/TopologyToggleIcon.scss

```diff
@@ -0,0 +1,4 @@
+.odc-topology-toggle-icon {
+  flex-grow: 1;
+  text-align: right;
+}
```

Comment:
Do we really not have a linter or something to catch no EOF line? Perhaps because this is `scss`. @christianvogt - do you have an opinion on this? If you're against not having the EOF, perhaps we find a way to lint it.

Reply:
style linting is still something we haven't done. I started it but never got it committed. i hate pointing out nits like this that a linter would catch. So i don't bother fighting those battles.